app/styles/courses/teacher-classes-view.sass

```diff
@@ -120,3 +120,46 @@
   .create-class
     margin-top: 65px
     margin-bottom: 65px
+
+  .center
+    text-align: center
+
+  .teacher-quests
+    border: 1px solid gray
+    margin: 20px 80px 40px 80px
+    padding: 10px
+    .quest-progress
+      font-size: 50px
+      margin: 20px 0px
+    .quests-summary-col
+    .quests-list-col
+      > ul
+        list-style: none
+        padding-left: 0
+        margin-left: 0
+      li.quest-complete
+        padding-left: 20px
+        font-size: 12px
+        opacity: 0.5
+        line-height: 14px
+        > div > p
+          margin-bottom: 7px
+        .quest-title::before
+          content: "☑ "
+          font-size: 200%
+      li.quest-incomplete
+        li
+          font-size: 15px
+        .quest-title::before
+          content: "☐ "
```

Comment:
If there's a way to use glyphicons for this, that would be preferable — locally in my editor, the ☑️ character (as used here) renders differently than ☑︎, and there's yet another, ☑ (which renders differently locally than in this editor right here).

I strongly suspect we'd get some weirdly inconsistent presentation of these characters on various browsers/OS's.